BHoMUpgrader40/Upgrades.json

```diff
@@ -1403,10 +1420,200 @@
+    "BH.Engine.Adapters.DIALux.Convert.FromDialUXPoint(System.String)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Adapters.DIALux.Convert.FromDialUXSpace(System.Collections.Generic.List<System.String>)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Adapters.DIALux.Convert.ToDIALux(BH.oM.Environment.Elements.Opening)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Adapters.DIALux.Convert.ToDialUX(BH.oM.Environment.Elements.OpeningType)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Adapters.DIALux.Convert.ToDialUX(BH.oM.Geometry.Point, System.Int32, System.Boolean)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Adapters.DIALux.Convert.ToDialUX(System.Collections.Generic.List<BH.oM.Environment.Elements.Panel>)": "This operation is now handled automatically by the DIALux Adapter and it is no longer necessary to perform this manually. As such, this method has been removed from UI use. If you need further assistance with this change, please raise an issue on https://github.com/BHoM/DIALux_Toolkit/issues",
+    "BH.Engine.Environment.Modify.UpdatePanelTypeByCustomData(System.Collections.Generic.List<BH.oM.Environment.Elements.Panel>, System.String)", "This method was providing a way of updating Panel Types based on a very specific Custom Data schema, which was not reliable in mainstream workflows. The method has been removed to avoid confusion. If further support is needed with setting the Panel Type, please raise an issue on https://github.com/BHoM/BHoM_Engine/issues"
```

Comment:
```suggestion
    "BH.Engine.Environment.Modify.UpdatePanelTypeByCustomData(System.Collections.Generic.List<BH.oM.Environment.Elements.Panel>, System.String)": "This method was providing a way of updating Panel Types based on a very specific Custom Data schema, which was not reliable in mainstream workflows. The method has been removed to avoid confusion. If further support is needed with setting the Panel Type, please raise an issue on https://github.com/BHoM/BHoM_Engine/issues"
```

Reply:
🤦‍♂️